README.md

```diff
@@ -1,61 +1,62 @@
 [![Build Status](https://dev.azure.com/knotx/Knotx/_apis/build/status/Knotx.knotx-starter-kit?branchName=master)](https://dev.azure.com/knotx/Knotx/_build/latest?definitionId=3&branchName=master)
 
 # Knot.x Starter Kit
-Knot.x Starter Kit is a template project that simplifies Knot.x project setup. It allows you to 
+Starter Kit is a template project that simplifies Knot.x project setup. It allows you to 
 customize the [Knot.x distribution](https://github.com/Knotx/knotx-stack) with your own modules and
-configuration entries. Then it builds and validates your custom Docker image. 
+configuration entries. 
 
+Starter Kit artefacts (see the build/distributions folder) are: 
+- a ZIP file when `./gradlew build-stack` or `./gradlew build`
+- a custom Docker image when `./gradlew build-docker`
 
-## How to build
+### Prerequisites
```

Comment:
What are those prerequisites?
It looks to me as it is for development -> shouldn't be it in that section?

Reply:
Knot.x Stack use Knot.x SNAPSHOT version by default. So you need all X.Y.Z-SNAPSHOT versions available. The same with Docker image.

Reply:
Fixed this comment. Explained in when talking about zipped Starter Kit versions.